### DISCUSSION TITLE
Telerik Removal in v10

### DISCUSSION THREAD

**Discussion by SkyeHoefling** [dnnsoftware/Dnn.Platform#3798] [General] [answered]
I was doing some research around how the HTML Editor works and came across some code that didn't make 100% sense to me. If I recall correctly from a few different posts it has been discussed that Telerik will be removed in v10 of DNN. I believe this will include `Telerik.Web.UI.dll`.

I do not see any issues created to refactor the HTML Editor Module which has a hard dependency on `RadToolbarButton` from the deprecated project.
https://github.com/dnnsoftware/Dnn.Platform/blob/8e172097c733654512be334d10a936f75e0ed27b/DNN%20Platform/Modules/HTML/HtmlModule.ascx#L8-L21

If we were to remove this dll in v10 I believe the HTML Editor would not allow any inline editing.

Is there an issue created for this? If not I can write something up

**Comment by mitchelsellers** [comment, marked as answer]
@ahoefling This is a good catch and should be documented

**Comment by SkyeHoefling** [comment]
Created #3800 to document this

**Comment by david-poindexter** [comment]
Thank you for catching this!

**Comment by sleupold** [comment]
One option to solve this issue would be using CKEditor 4 in inline mode

**Reply by thabaum** [reply]
I like the idea of inline as it would be also nice to have both like before.

**Comment by thabaum** [comment]
https://ckeditor.com/ckeditor-5/ would updating to newer CK editor 5 and make adjustments while doing this as needed be something of interest to do?

**Reply by mitchelsellers** [reply]
Interest, yes, but it is an area that does not need touched during the current critical path of removing Telerik

**Reply by mitchelsellers** [reply]
Additionally if any breaking changes would need to hold for 10.0.0
